addons/web_editor/static/src/js/wysiwyg/plugin/keyboard.js

```diff
@@ -900,6 +947,33 @@ var KeyboardPlugin = AbstractPlugin.extend({
         }
         return range;
     },
+    /**
+     * Select all the contents of the current unbreakable ancestor.
+     */
+    _selectAll: function () {
+        var range = this.context.invoke('editor.createRange');
+        var unbreakable = dom.ancestor(range.sc, this.options.isUnbreakableNode);
+        var $contents = $(unbreakable).contents();
+        var pointA = {
+            node: $contents.length ? $contents[0] : unbreakable,
+            offset: 0,
+        };
+        pointA = dom.nextPointUntil(pointA, function (point) {
+            return point.node && !(dom.isText(point.node) && /^\s*$/.test(point.node.textContent));
+        });
+        var pointB = {
+            node: $contents.length ? $contents[$contents.length - 1] : unbreakable,
+            offset: $contents.length ? dom.nodeLength($contents[$contents.length - 1]) : 1,
+        };
+        pointA = dom.prevPointUntil(pointA, function (point) {
+            return point.node && !(dom.isText(point.node) && /^\s*$/.test(point.node.textContent));
```

Comment:
Possible/interesting to extract to an helper function?
Or reuse something like `HelperPlugin.getRegexBlank()`?

Reply:
Good point again, there actually already was a function I could use : `HelperPlugin.isVisibleText`...
Also I noticed there that I'm moving `pointA` instead of `pointB` so this was effectively doing nothing. Fixed.